pkg/cmd/policy_validate.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/sdk/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyValidateCmd() *cobra.Command {
+	var argConfig string
+
+	var cmd = &cobra.Command{
+		Use:   "validate <org-name>/<policy-pack-name> <version>",
```

Comment:
We should support "latest" for this like we do for the enable command

Reply:
I had tried supporting latest, but api call was failing. From what I can tell, it's because the api isnt configured to support latest in the call path `api/orgs/{orgName}/policypacks/{policyPackName}/versions/{version}/schema`. When sending latest the call just came back saying it couldn't find the version. So I think this may involve some work on service side to support this functionality.....

Reply:
ah yes this handler is def not implemented.. kk lets not worry about that for now then (if we get a request then we can add it)